includes/watchers/class-algolia-post-changes-watcher.php

```diff
@@ -39,10 +44,15 @@ public function watch() {
 	 * @param int $post_id
 	 */
 	public function sync_item( $post_id ) {
+
 		if ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) {
 			return;
 		}
 
+		if(in_array($post_id, $this->postsDeleted)) {
```

Comment:
Put spaces on both sides of the opening and closing parentheses of `if`, and within the `in_array` function call per [WordPress standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage):
```
if ( in_array( $post_id, $this->postsDeleted ) ) {
```